source/shared/cpp/ObjectModel/ChoicesData.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "pch.h"
+#include "ParseContext.h"
+
+namespace AdaptiveCards
+{
+class ChoicesData
+{
+public:
+    ChoicesData();
+    ChoicesData(std::string type, std::string dataset) :
+        m_type(type), m_dataset(dataset)
+    {
+    }
+
+    bool ShouldSerialize() const;
+    std::string Serialize() const;
+    Json::Value SerializeToJsonValue() const;
+
+    std::string GetType() const;
```

Comment:
can you change the name to something more distinguishing since we already use type for AdaptiveElement?

Reply:
done